src/tools/schema2kotlin.ts

```diff
@@ -205,8 +252,26 @@ Schema(
         singletons = ${leftPad(ktUtils.mapOf(this.singletonSchemaFields, 30), 8, true)},
         collections = ${leftPad(ktUtils.mapOf(this.collectionSchemaFields, 30), 8, true)}
     ),
-    "${schemaHash}"
-)`;
+    "${schemaHash}",
```

Comment:
This is my bad: here, let's use the `quote` function.

Reply:
Also, for these arguments: if refinement and query are named arguments, maybe hash (and others?) should be as well. Or, all of them should be unnamed.

Reply:
Quote does the exact same thing. Should it be escaping special characters (to be general)?

Reply:
Also, yep, we should probably name all the arguments to make the generated code more likely to be backwards compatible, but I'm going to leave it for now.